rasa/core/policies/keras_policy.py

```diff
@@ -298,26 +287,16 @@ def load(cls, path: Text) -> "KerasPolicy":
             if os.path.isfile(meta_file):
                 meta = json.loads(rasa.utils.io.read_file(meta_file))
 
-                tf_config_file = os.path.join(path, "keras_policy.tf_config.pkl")
-                with open(tf_config_file, "rb") as f:
-                    _tf_config = pickle.load(f)
-
                 model_file = os.path.join(path, meta["model"])
 
-                graph = tf.Graph()
-                with graph.as_default():
-                    session = tf.compat.v1.Session(config=_tf_config)
-                    with session.as_default():
-                        with warnings.catch_warnings():
-                            warnings.simplefilter("ignore")
-                            model = load_model(model_file)
+                with warnings.catch_warnings():
```

Comment:
which kind of warnings are we catching here? Probably makes sense to add a comment.

Reply:
Not sure about that.